sdk/monitor/azingest/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,76 @@
+# Troubleshooting Azure Monitor Ingestion module
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Monitor Ingestion module for Go.
+
+## Table of contents
+
+* [General troubleshooting](#general-troubleshooting)
+    * [Error Handling](#error-handling)
+    * [Logging](#logging)
+* [Troubleshooting logs ingestion](#troubleshooting-logs-ingestion)
+    * [Troubleshooting authorization errors](#troubleshooting-authorization-errors)
+    * [Troubleshooting missing logs](#troubleshooting-missing-logs)
+    * [Troubleshooting slow logs upload](#troubleshooting-slow-logs-upload)
+
+## General troubleshooting
+
+### Error Handling
+
+All methods which send HTTP requests return `*azcore.ResponseError` when these requests fail. `ResponseError` has error details and the raw response from Monitor Ingestion.
+
+### Logging
+
+To troubleshoot issues with the module, first enable logging to monitor the behavior of the application. The errors and warnings in the logs generally provide useful insights into what went wrong and sometimes include corrective actions to fix issues.
+
+This module uses the logging implementation in `azcore`. To turn on logging for all Azure SDK modules, set `AZURE_SDK_GO_LOGGING` to `all`. By default, the logger writes to stderr. Use the `azcore/log` package to control log output. For example, logging only HTTP request and response events, and printing them to stdout:
```

Comment:
Could call the specific log flags which would be most helpful to debug monitor issues.

Reply:
Don't have specific log flags for monitor ingestion, so the HTTP reuest and response events are the most relevant.